build/msw/makefile.gcc

```diff
@@ -5261,109 +5266,76 @@ clean:
 setup_h: $(SETUPHDIR)\wx ..\..\include\wx\$(__SETUP_H_SUBDIR_FILENAMES)\setup.h $(SETUPHDIR)\wx\setup.h $(SETUPHDIR)\wx\msw\rcdefs.h
 
 $(LIBDIRNAME)\libwxregex$(WXUNICODEFLAG)$(WXDEBUGFLAG).a: $(WXREGEX_OBJECTS)
-	$(foreach f,$(subst \,/,$(WXREGEX_OBJECTS)),$(shell echo $f >> $(subst \,/,$@).rsp.tmp))
```

Comment:
Lots of unrelated changes, because the bakefile version used to generate these files is newer than `0.2.11`, it now uses response files.

You could change the files of your bakefile installation with [these](https://github.com/vslavik/bakefile/commit/8e8bfbfd49331d1957f12ae549a655200fefb6a7) changes. Or maybe just let @vadz rebake before merging.